recipes/python-leveldb/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.193" %}
+
+package:
+  name: python-leveldb
+  version: {{ version }}
+
+source:
+  fn: leveldb-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/l/leveldb/leveldb-{{ version }}.tar.gz
+  md5: 2952434f2a0ce10c44f58542cc561589
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - leveldb
+    - python
+    - setuptools
+
+  run:
+    - leveldb
+    - python
+
+test:
+  imports:
+    - leveldb
+
+about:
+  home: http://code.google.com/p/py-leveldb/
```

Comment:
Unfortunately, they still seem to be using the Google Code site. So, that's what we have. At some point, they will move and we can update.